pkg/apic/specasyncapiprocessor.go

```diff
@@ -32,6 +33,20 @@ func (p *asyncAPIProcessor) GetVersion() string {
 	return ""
 }
 
+// GetDescription -
+func (p *asyncAPIProcessor) GetDescription() string {
+	info := p.asyncapiDef["info"]
+	if info != nil {
+		if infoDetail, ok := info.(map[string]interface{}); ok {
+			version := infoDetail["description"]
```

Comment:
wanna change version var to description, just to match?  or if its really a version description, then something like versionDesc?